packages/terra-framework-docs/src/terra-dev-site/doc/compact-interactive-list/Examples.2/FlowDirection.4.doc.mdx

```diff
@@ -0,0 +1,27 @@
+import VerticalFlow from './VerticalFlow?dev-site-examples';
+import HorizontalFlow from './HorizontalFlow?dev-site-examples';
+
+# Flow Direction
+
+### Vertical Flow
+
+By default the items in the [Compact Interactive List](/components/cerner-terra-framework-docs/compact-interactive-list/about) go from top to bottom before breaking into a new visual column.
+
+<VerticalFlow />
+
+#### Usage
+
+* The list items needs the [_rowHeight_](/components/cerner-terra-framework-docs/compact-interactive-list/about#compact-interactive-list-props-table) property to be explicitly set for the vertical flow. If not set by user, the default value will be used.
+* The items will be distributed evenly between all the columns but the last one. The last column will accommodate the reminder after the number of items was divided by number of columns.
```

Comment:
Should this be "remainder"?
```suggestion
* The items will be distributed evenly between all the columns but the last one. The last column will accommodate the remainder after the number of items was divided by number of columns.
```

Reply:
Great suggestion, and much appreciated, but it looks like all wording and styling comment were outdated per this commit by @cldeitrich: https://github.com/cerner/terra-framework/pull/1950/commits/62c72c0098e6edb05ec923b22cf997650ac247bf